ephios/core/signup/methods.py

```diff
@@ -534,22 +536,41 @@ def render_configuration_form(self, *args, form=None, **kwargs):
 @dataclasses.dataclass()
 class SignupStats:
     requested_count: int
-    signed_up_count: int
-    missing: Optional[int]
-    free: Optional[int]
+    confirmed_count: int
+    missing: int
```

Comment:
why isn't missing optional anymore? if I don't have a minimun, I don't have a missing count

Reply:
So zero people are missing?!

Reply:
yeah just thought about the label in the event list, but we check for > 0 there anyways.